lib/active_data.rb

```diff
@@ -43,6 +43,12 @@ def self.config
     ActiveData::Config.instance
   end
 
+  def self.legacy_active_model?
```

Comment:
@rewritten Maybe pass the definition of legacy as argument to this method.

When AM 7.0 will be released how would you scale this method if the definition of legacy is hardcoded?

Reply:
@rewritten I didn't get what you mean here. Do you have an example?